### PR TITLE
[brynet] Update to 1.0.5

### PR DESCRIPTION
**Pull request by IronsDu**
update brynet to tag 1.0.5 .(after now, brynet changed to header only)

### CODE REVIEW

File: ports/brynet/portfile.cmake [hunk 1]
```diff
@@ -1,23 +1,23 @@
+#header-only library
 include(vcpkg_common_functions)
```
Comment (PhoebeHui) [inline ports/brynet/portfile.cmake]: Could you please remove 'include(vcpkg_common_functions)', this is deprecated.

File: ports/brynet/portfile.cmake [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+        -DCMAKE_INSTALL_DIR:STRING=cmake
 )
-
 vcpkg_install_cmake()
 
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
-
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/lib ${CURRENT_PACKAGES_DIR}/debug)
 file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/brynet)
 file(RENAME ${CURRENT_PACKAGES_DIR}/share/brynet/LICENSE ${CURRENT_PACKAGES_DIR}/share/brynet/copyright)
```
Comment (PhoebeHui) [inline ports/brynet/portfile.cmake]: Could you update 
file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/brynet)
file(RENAME ${CURRENT_PACKAGES_DIR}/share/brynet/LICENSE ${CURRENT_PACKAGES_DIR}/share/brynet/copyright)

to 
file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

File: ports/brynet/portfile.cmake [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
     PREFER_NINJA
+    OPTIONS
+        -DCMAKE_INSTALL_DIR:STRING=cmake
 )
-
 vcpkg_install_cmake()
 
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
-
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/lib ${CURRENT_PACKAGES_DIR}/debug)
```
Comment (PhoebeHui) [inline ports/brynet/portfile.cmake]: If you want to change this port to header only, you can remove these lines and use 'file(INSTALL ${SOURCE_PATH}/include/brynet DESTINATION ${CURRENT_PACKAGES_DIR}/include)' instead.

**Review by PhoebeHui — COMMENTED** [review]
Could you also update below information?
1. Could you also update CONTROL file?
2. What issue does this PR fix? Please open an issue so others may replicate this problem.
3. Please update the baseline CI file https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt#L176

File: ports/brynet/portfile.cmake (hunk 3 above)
Comment (IronsDu) [reply ports/brynet/portfile.cmake]: Fix it, thank you very much!

**Comment by PhoebeHui** [comment]
@IronsDu, could you remove below lines in baseline CI file https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt#L176?

brynet:arm-uwp=fail
brynet:x64-linux=fail
brynet:x64-osx=fail
brynet:x64-uwp=fail

**Comment by IronsDu** [comment]
> @IronsDu, could you remove below lines in baseline CI file https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt#L176?
> 
> brynet:arm-uwp=fail
> brynet:x64-linux=fail
> brynet:x64-osx=fail
> brynet:x64-uwp=fail

Thank you.  I already remoe this lines.

**Review by PhoebeHui — APPROVED** [review]

**Comment by IronsDu** [comment]
@PhoebeHui @dan-shaw Excause me, can you merge this pr? Thanks.

**Review by dan-shaw — APPROVED** [review]

**Comment by dan-shaw** [comment]
The MacOS CI pipeline is still down for us. Can you change the baseline file back to `brynet:x64-osx=fail`?  After our pipeline is back up, we can have another PR that changes it to passing. Otherwise, I'm OK with merging this.

**Comment by IronsDu** [comment]
> The MacOS CI pipeline is still down for us. Can you change the baseline file back to `brynet:x64-osx=fail`? After our pipeline is back up, we can have another PR that changes it to passing. Otherwise, I'm OK with merging this.

Hi, I see 'oxs:fail' already removed in this commit : https://github.com/microsoft/vcpkg/pull/9277/commits/25ad5293bd17adb88983a831bad8af20e1217f35, But vcpkg-osx-PR Pending — queued, I don't know it's why.
I think you can merge this pr . ^_^. Thank you.

**Pull request merged**